stencil-library/src/components/details/details.tsx

```diff
@@ -0,0 +1,97 @@
+import { Component, Host, h, Prop } from '@stencil/core';
+import { ErrorState } from './utils';
+
+interface IEntityProps {
+  title: string;
+  entityHeadInfo: {
+    title: string,
+    value: any
+  }[];
+  entitySections: {
+    sectionTitle: string;
+    sectionDetails: {
+      title: string,
+      value: any
+    }[];
+  }[];
+  metadata: any;
+}
+
+@Component({
+  tag: 'justifi-details',
+  styleUrl: './details.scss',
+  shadow: true,
+})
+
+export class Details {
+  @Prop() errorMessage: string;
+  @Prop() entity: IEntityProps;
+
+  render() {
+    return (
+      <Host
+        exportParts='detail-loading-spinner,detail-loading-state,detail-empty-state,
+        detail-head,detail-title,detail-method,detail-info,detail-info-item,
+        detail-info-item-title,detail-info-item-data,detail-metadata,detail-metadata-title,
+        detail-method-title,detail-method-data'
+      >
+        {
+        this.errorMessage ?
+          ErrorState(this.errorMessage)
+        :
+          <main class="p-2">
+            <div part='detail-head' class="p-2">
+              <div class="d-flex flex-row align-items-center gap-2 mb-2">
+                <h1 class="m-0 text-uppercase" part='detail-title'>{this.entity.title}</h1>
+                <slot name="badge" />
+              </div>
+              <div part="detail-head-info" class="d-flex flex-row align-items-top">
+                {
+                  this.entity.entityHeadInfo.map((item, index) =>
+                    <div part="detail-head-info-item"
+                      class={`d-flex flex-column ${index === this.entity.entityHeadInfo.length-1 ? '' : 'border-1 border-end'}`}
+                    >
+                      <span part="detail-head-info-item-title" class="fw-bold border-1 border-bottom ps-2 pe-2">{item.title}</span>
+                      <span part="detail-head-info-item-data" class="pt-2 ps-2 pe-2">{item.value}</span>
+                    </div>
+                  )
+                }
+              </div>
+            </div>
+            {
+              this.entity.entitySections.map(section =>
+                <div part="detail-details" class="p-2 mt-4">
+                  <h2 part="detail-details-title" class="fw-bold fs-1">{section.sectionTitle}</h2>
```

Comment:
I believe in the mocks these are `h5`s

Reply:
I may be misreading the Figma, but it says H2 there:
<img width="1216" alt="image" src="https://github.com/justifi-tech/web-component-library/assets/10523683/7bab71ae-91b1-47e1-9939-e9475eff50b3">

Reply:
Oh, I think I may be looking at an older version.